qemu/tests/cfg/boot_from_device.cfg

```diff
@@ -76,9 +76,10 @@
         - boot_from_iscsi_device:
             start_vm = no
             dev_name = iscsi-dev
-            portal_ip = 10.66.90.100
-            initiator = "iqn.2010-07.com.redhat:kvmautotest"
-            target = "iqn.2001-05.com.equallogic:0-8a0906-db31f7d03-470263b05654c204-kvm-puyiqiao"
+            # Please update these to point to your iscsi server
+            portal_ip = 127.0.0.1
+            initiator = "iqn.2010-07.com.example:my_initiator"
+            target = "iqn.2001-05.com.mytarget:0-0a0000-000000000-0000000000000000-my-target"
```

Comment:
Does including those first 4 patches in your PR means they are ACKed by you? If so we can merge https://github.com/autotest/tp-qemu/pull/1175 as the shas are the same and @luckyh already gave it first ACK.